Sources/MongoSwift/Operations/ListDatabasesOperation.swift

```diff
@@ -30,9 +30,17 @@ public struct ListDatabasesOptions {
     /// Specifies whether to only return databases for which the user has privileges.
     public var authorizedDatabases: Bool?
 
+    /// An arbitrary BSON type to help trace the operation through
+    /// the database profiler, currentOp and logs. The default is to not send a value.
+    public var comment: BSON?
```

Comment:
ditto add a test

Reply:
Added `testListDatabasesComment()` to `MongoClientTests`

Reply:
had to use a workaround to test server version since it looked like i couldnt access `serverVersionIsInRange()` from `MongoClientTests`, presumably because im testing the client itself (so had to use a more manual workaround)